types/evidence.go

```diff
@@ -681,6 +681,16 @@ func (evl EvidenceList) Has(evidence Evidence) bool {
 	return false
 }
 
+// ToABCI converts the evidence list to a list of the equivalent abci proto type
+// for use when communicating the evidence to an application.
+func (evl EvidenceList) ToABCI() []abci.Evidence {
+	el := make([]abci.Evidence, 0)
+	for _, e := range evl {
+		el = append(el, e.ABCI()...)
+	}
+	return el
```

Comment:
```suggestion
	out := make([]abci.Evidence, len(evl))
	for i, e := range evl {
		out[i] = e.ABCI()
	}
	return out
```

Rationale: If we're going to preallocate, it should be to the full length, and then there's no real need to append.